scripts/deploy.sh

```diff
@@ -51,6 +52,11 @@ ansible-galaxy install --role-file=/opt/rpc-openstack/ansible-role-requirements.
 # Enable playbook callbacks from OSA to display playbook statistics
 grep -q callback_plugins playbooks/ansible.cfg || sed -i '/\[defaults\]/a callback_plugins = plugins/callbacks' playbooks/ansible.cfg
 
+#Clone Magnum repository if required
+if [[ "$DEPLOY_MAGNUM" == "yes" ]]; then
+  git clone https://github.com/openstack/openstack-ansible-os_magnum.git -b stable/newton --single-branch $OA_DIR/playbooks/roles/os_magnum
```

Comment:
> And where would that role be installed? I'll need to reference the path for some of the later portions.

There's a role path that ansible uses. It's typical /etc/ansible/roles. Why do you think you'll need to reference the path to the role? If it's for the extras and secrets bits, I'd tend to agree with @odyssey4me that those should live in this repository rather than trying to copy them out of the git clone